views/Geographic/GeographicMap.svelte

```diff
@@ -13,20 +15,74 @@
   export let contentWidth;
   export let baseMap;
   export let formattingOptions;
+  export let annotations = [];
+  
+  let cssModifier = getCssModifier(contentWidth);
 
   const maxHeight = 550;
   const geoParameters = getGeoParameters(baseMap, contentWidth, maxHeight);
   const bounds = geoParameters.bounds;
-  const height = round(bounds[1][1]);
-  const viewBox = [bounds[0][0], bounds[0][1], bounds[1][0], bounds[1][1]]
-    .map(value => round(value))
-    .join(" ");
+
+  // Constants for annotations
+  const annotationRadius = 9;
+  const annotationStartPosition = 16;
+  const lineStartPosition = annotationStartPosition - annotationRadius;
+  
+  const svgSize = getSvgSize(bounds, annotations, annotationRadius, annotationStartPosition); 
+
+  annotations = setCoordinatesForGeoMap(annotations, geoParameters, entityType, annotationStartPosition, lineStartPosition, cssModifier);
+
+  function getCssModifier(contentWidth) {
+    if (contentWidth < 400) {
+      return "narrow";
+    } else if (contentWidth < 470) {
+      return "wide";
+    } else if (contentWidth < 650) {
+      return "wide-plus";
+    } else {
+      return "extra-wide";
+    }
+  }
+  
+  function getSvgSize(bounds, annotations, annotationRadius, annotationStartPosition) {
```

Comment:
Maybe therre's a way to abstract both `getSvgSize` functions and put in the helper-file? The functionality seems oddly similar.

Reply:
I moved part of the function to a helper file. Should be easier to read now.